odoo/models.py

```diff
@@ -6230,14 +6281,14 @@ def process(node, info, prefix):
                 if not result.get(names):
                     result[names] = node.attrib.get('on_change')
                 # traverse the subviews included in relational fields
-                for subinfo in info['fields'][name].get('views', {}).values():
-                    process(etree.fromstring(subinfo['arch']), subinfo, names)
+                for child_view in node.xpath("./*[descendant::field]"):
+                    process(child_view, None, names)
```

Comment:
This is a bit confusing.  Aren't the subviews removed from the view itself?  Or did you change that behavior?

Reply:
I didn't read the PR description, sorry :frowning_face: